lib/src/mime_type.dart

```diff
@@ -25,6 +25,19 @@ int get defaultMagicNumbersMaxLength => _globalResolver.magicNumbersMaxLength;
 String lookupMimeType(String path, {List<int> headerBytes}) =>
     _globalResolver.lookup(path, headerBytes: headerBytes);
 
+/// Returns the extension for the given MIME type.
+///
+/// If there are multiple extensions for [mime], return the first occurrence in
+/// the map. If there are no extensions for [mime], return [mime].
+String extensionFromMime(String mime) {
+  for (final key in defaultExtensionMap.keys) {
+    if (defaultExtensionMap[key].toLowerCase() == mime.toLowerCase()) {
```

Comment:
consider doing `mime = mime.toLowerCase()` at the top of the function – convert once.

Also consider just adding a test that validates all of the values in the map `value == value.toLowerCase` then you can avoid doing the conversion for each value

Reply:
done